src/pages/open-source/index.tsx

```diff
@@ -0,0 +1,234 @@
+                                                data-size="large"
+                                                data-show-count="true"
+                                                aria-label="Star apify/crawlee on GitHub"
+                                            >
+                                                Star
+                                            </GitButton>
+                                        </div>
+                                    </div>
+                                }
+                            />
+                            <CardWithImageAndContent
+                                content={
+                                    <div className="cardContentWrapper">
+                                        <div className="cardContentWrapperText">
+                                            <Link to="https://github.com/apify/proxy-chain" className="otherCardsTitleLink">
+                                                <Heading type="titleXl" as="h3">
+                                                    Proxy chain
+                                                </Heading>
+                                            </Link>
+                                            <Text color={theme.color.neutral.textMuted} size="big">
+                                                A Node.js proxy server with support for SSL, authentication, upstream proxy chaining, custom HTTP responses, and
+                                                traffic statistics.
+                                            </Text>
+                                        </div>
+                                        <div>
+                                            <GitButton
+                                                href="https://github.com/apify/proxy-chain"
+                                                data-size="large"
+                                                data-show-count="true"
+                                                aria-label="Star apify/crawlee on GitHub"
+                                            >
+                                                Star
+                                            </GitButton>
+                                        </div>
+                                    </div>
+                                }
+                            />
+                        </div>
+                    </Section>
+                    <Section
+                        heading="Actor templates"
+                        description="Actor templates help you quickly set up your web scraping projects.
+                            Save development time and get immediate access to all the features that the Apify platform has to offer."
+                        headingAs="h2"
+                    >
+                        <div className="actorTemplates">
+                            <ActorTemplates displayedTemplatesIds={['python-start', 'js-start', 'ts-start']} />
```

Comment:
Nit: If you want, we have a better icon for dark theme for HTTPX here: https://github.com/apify/apify-core/blob/develop/src/console/frontend/public/img/logos_other/logo_httpx.svg